demo/kubernetes/rook-operator.yaml

```diff
@@ -12,5 +12,9 @@ spec:
     spec:
       containers:
       - name: rook-operator
-        image: quay.io/rook/rook-operator:dev-2017-02-17-k8s
-        args: ["--container-version=dev-2017-02-17-k8s"]
+        image: quay.io/rook/rook-operator:dev-2017-02-23-k8s
+        env:
+        - name: ROOK_OPERATOR_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

Comment:
I'm not sure I follow this. So the operator can run in a namespace, and the cluster it creates can be in a different namespace?